clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesCacheUtils.java

```diff
@@ -46,7 +45,7 @@
 
 @Component
 @Slf4j
-public class KubernetesCacheUtils {
+class KubernetesCacheUtils {
```

Comment:
Given all the work I've been doing with CRD extensions lately I feel like we might actually want want to leave this public so it can be used in any custom caching agents. Thoughts?

Reply:
In general when people write custom caching agents, what package do they put them in?  I'd also be interested in how people are using this in custom caching agents if none of the built-in ones use it.

Reply:
(The reason I ask is that classes like this that become implicit public contracts are effectively impossible to refactor if we don't have any boundaries on how people are using them in custom builds.)

Reply:
I actually haven't seen anyone using a custom caching agent. I'd assume they put it in a package alongside their custom handler. For example, we use `io.armory.spinnaker` for a lot of our internal extensions. If it were me, I'd probably put it under something like `io.armory.spinnaker.kubernetes.{handler, cache}`. We should probably ask some of the users we know have custom handlers to be sure. 

Totally understand the need to make extension points more explicit. Just want to be sure we don't break anyone in the process w/o warning.

Reply:
I pushed a commit that un-does making `KubernetesCacheUtils` package-private to not hold this up while seeing if that will break anyone.  I'll reach out to a couple of users that I know have build custom handlers to see if they use this.